app/views/registrations/index.html.erb

```diff
@@ -1,5 +1,7 @@
 <% provide(:title, I18n.t('registrations.list.title', comp: @competition.name)) %>
 
 <%= render layout: "nav" do %>
-  <%= react_component('RegistrationsV2/Registrations', { competitionInfo: @competition.to_competition_info }) %>
+  <%= react_component('RegistrationsV2/Registrations', { competitionInfo: @competition.to_competition_info, userInfo: @current_user.as_json(
+    only: %w[id],
+  )}) %>
```

Comment:
Now if you really only need the user's ID, you can just pass it as `userId` prop, no?